From 566a085f26168bd391f2aa4d6d189907a9796a70 Mon Sep 17 00:00:00 2001
From: Arne Coucheron <arco68@gmail.com>
Date: Thu, 17 Aug 2017 01:19:05 +0200
Subject: [PATCH 158/194] Snap: Update HDR icons

Current ones are so small, I can hardly see them. These new ones
are bigger and looks more in line with the rest of the icons.

Change-Id: Idaeec67a78d66105a375f09b6701a49dce43fd22
---
 res/drawable/ic_hdr.xml            | 15 ++++++++++-----
 res/drawable/ic_hdr_off.xml        | 15 +++++++++------
 res/drawable/ic_scene_mode_hdr.xml | 15 ++++++++++-----
 3 files changed, 29 insertions(+), 16 deletions(-)

diff --git a/res/drawable/ic_hdr.xml b/res/drawable/ic_hdr.xml
index 90c7f847c..63e9e3438 100644
--- a/res/drawable/ic_hdr.xml
+++ b/res/drawable/ic_hdr.xml
@@ -2,12 +2,17 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="24dp"
     android:height="24dp"
-    android:viewportWidth="24"
-    android:viewportHeight="24">
+    android:viewportWidth="48"
+    android:viewportHeight="48">
 
+    <path
+        android:pathData="M0 0h48v48h-48z" />
     <path
         android:fillColor="@color/white"
-        android:pathData="M21 11.5v-1c0-.8-.7-1.5-1.5-1.5H16v6h1.5v-2h1.1l.9 2H21l-.9-2.1c.5-.3 .9 -.8 .9
--1.4zm-1.5 0h-2v-1h2v1zm-13-.5h-2V9H3v6h1.5v-2.5h2V15H8V9H6.5v2zM13
-9H9.5v6H13c.8 0 1.5-.7 1.5-1.5v-3c0-.8-.7-1.5-1.5-1.5zm0 4.5h-2v-3h2v3z" />
+        android:pathData="M12 22h-4v-8h-4v20h4v-8h4v8h4v-20h-4v8zm14-8h-8v20h8c2.21 0 4-1.79
+4-4v-12c0-2.21-1.79-4-4-4zm0
+16h-4v-12h4v12zm18-8v-4c0-2.21-1.79-4-4-4h-8v20h4v-8h2l2 8h4l-2.38-8.35c1.4-.62
+2.38-2.02 2.38-3.65zm-4 0h-4v-4h4v4z" />
+    <path
+        android:pathData="M0 0h48v48h-48z" />
 </vector>
diff --git a/res/drawable/ic_hdr_off.xml b/res/drawable/ic_hdr_off.xml
index 53a6c78ad..7e1d8be23 100644
--- a/res/drawable/ic_hdr_off.xml
+++ b/res/drawable/ic_hdr_off.xml
@@ -2,13 +2,16 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="24dp"
     android:height="24dp"
-    android:viewportWidth="24"
-    android:viewportHeight="24">
+    android:viewportWidth="48"
+    android:viewportHeight="48">
 
+    <path
+        android:pathData="M0 0h48v48h-48zm0 0h48v48h-48z" />
     <path
         android:fillColor="@color/white"
-        android:pathData="M17.5 15v-2h1.1l.9 2H21l-.9-2.1c.5-.2 .9 -.8 .9
--1.4v-1c0-.8-.7-1.5-1.5-1.5H16v4.9l1.1 1.1h.4zm0-4.5h2v1h-2v-1zm-4.5 0v.4l1.5
-1.5v-1.9c0-.8-.7-1.5-1.5-1.5h-1.9l1.5 1.5h.4zm-3.5-1l-7-7-1.1 1L6.9
-9h-.4v2h-2V9H3v6h1.5v-2.5h2V15H8v-4.9l1.5 1.5V15h3.4l7.6 7.6 1.1-1.1-12.1-12z" />
+        android:pathData="M36 34l-29.46-29.45-2.54 2.54 8 8v6.91h-4v-8h-4v20h4v-8h4v8h4v-14.91l2
+2v12.91h8c1.34 0 2.51-.66 3.24-1.67l12.67 12.67
+2.54-2.55-8.45-8.45zm-10-4h-4v-4.91l4 4v.91zm10-4h2l1.64 6.54 1.45
+1.46h2.91l-2.38-8.35c1.4-.62 2.38-2.02 2.38-3.65v-4c0-2.21-1.79-4-4-4h-8v10.91l4
+4v-2.91zm0-8h4v4h-4v-4zm-6 4.91v-4.91c0-2.21-1.79-4-4-4h-4.91l8.91 8.91z" />
 </vector>
diff --git a/res/drawable/ic_scene_mode_hdr.xml b/res/drawable/ic_scene_mode_hdr.xml
index 90c7f847c..63e9e3438 100644
--- a/res/drawable/ic_scene_mode_hdr.xml
+++ b/res/drawable/ic_scene_mode_hdr.xml
@@ -2,12 +2,17 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="24dp"
     android:height="24dp"
-    android:viewportWidth="24"
-    android:viewportHeight="24">
+    android:viewportWidth="48"
+    android:viewportHeight="48">
 
+    <path
+        android:pathData="M0 0h48v48h-48z" />
     <path
         android:fillColor="@color/white"
-        android:pathData="M21 11.5v-1c0-.8-.7-1.5-1.5-1.5H16v6h1.5v-2h1.1l.9 2H21l-.9-2.1c.5-.3 .9 -.8 .9
--1.4zm-1.5 0h-2v-1h2v1zm-13-.5h-2V9H3v6h1.5v-2.5h2V15H8V9H6.5v2zM13
-9H9.5v6H13c.8 0 1.5-.7 1.5-1.5v-3c0-.8-.7-1.5-1.5-1.5zm0 4.5h-2v-3h2v3z" />
+        android:pathData="M12 22h-4v-8h-4v20h4v-8h4v8h4v-20h-4v8zm14-8h-8v20h8c2.21 0 4-1.79
+4-4v-12c0-2.21-1.79-4-4-4zm0
+16h-4v-12h4v12zm18-8v-4c0-2.21-1.79-4-4-4h-8v20h4v-8h2l2 8h4l-2.38-8.35c1.4-.62
+2.38-2.02 2.38-3.65zm-4 0h-4v-4h4v4z" />
+    <path
+        android:pathData="M0 0h48v48h-48z" />
 </vector>
-- 
2.17.1

